weave/artifact_wandb.py

```diff
@@ -503,9 +516,14 @@ def add_artifact_reference(self, uri: str) -> None:
         # The URI for a get node might look like: wandb-artifact:///<entity>/<project>/test8:latest/obj
         # but the SDK add_reference call requires something like:
         # wandb-artifact://41727469666163743a353432353830303535/obj.object.json
-        assert uri.startswith("wandb-artifact:///")
-        uri_parts = WeaveWBArtifactURI.parse(uri)
-        uri_path = f"{uri_parts.entity_name}/{uri_parts.project_name}/{uri_parts.name}:{uri_parts.version}"
+        assert uri.startswith("wandb-artifact:///") or uri.startswith(
```

Comment:
This code reads quite odd now. I think this block should be reverted and instead put a single check in the beginning of the function:
```
if uri.startswith("wandb-artifact-by-id:///"):
   raise errors.WeaveInternalError("`add_artifact_reference` does not support `wandb-artifact-by-id` URI scheme. This is an internal programming error")
```